plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/incremental/IncrementalAptCache.kt

```diff
@@ -10,16 +10,19 @@ import java.io.Serializable
 
 class IncrementalAptCache : Serializable {
 
-    private val aggregatingGenerated: MutableMap<File, String?> = mutableMapOf()
+    private val aggregatingGenerated: MutableSet<File> = mutableSetOf()
     private val aggregatedTypes: MutableSet<String> = linkedSetOf()
-    private val isolatingMapping: MutableMap<File, Pair<String?, File>> = mutableMapOf()
-    // Annotations claimed by aggregating annotation processors
-    private val aggregatingClaimedAnnotations: MutableSet<String> = mutableSetOf()
+    private val isolatingMapping: MutableMap<File, String> = mutableMapOf()
 
     var isIncremental = true
         private set
 
-    fun updateCache(processors: List<IncrementalProcessor>): Boolean {
+    fun updateCache(processors: List<IncrementalProcessor>, failedToAnalyzeSources: Boolean): Boolean {
+        if (failedToAnalyzeSources) {
+            invalidateCache()
```

Comment:
Will it imply that next execution would be full rebuild and there wouldn't be any attempt to perform IC?